conceptnet5/vectors/query.py

```diff
@@ -187,6 +196,12 @@ def expanded_vector(self, terms, limit_per_term=10, include_neighbors=True):
             self.expand_terms(terms, limit_per_term, include_neighbors)
         )
 
+    def get_fasttext_vector(self, query):
+        """Look up a vector for a query in the fastText model."""
+        label = query.split('/')[-1]
```

Comment:
Can we be sure that the last segment of the term we're looking up is the text? There are URIs of the form `/c/en/thing/n`. We don't have vectors for them, though -- they're supposed to be reduced to `/c/en/thing` before we look them up.

In other methods I think we just wouldn't get a vector if we looked up one of those at this point. Here it would look up the word `n` in the fastText data. This seems like a more unclear failure mode.

Also, if we're looking up text in a raw fastText model that has never seen our URIs, where are the slashes coming from at all?